modules/client-configuration/pages/contact-methods-saltssh.adoc

```diff
@@ -51,15 +44,54 @@ Additionally, the required Python version will be installed with the salt-bundle
 
 
 
-== Available Parameters
+== Available parameters
 
 When you are configuring Push via Salt SSH, you can modify parameters that are used when a system is registered, including the host, activation key, and password.
 The password is used only for bootstrapping, it is not saved anywhere.
 All future SSH sessions are authorized via a key/certificate pair.
 These parameters are configured in menu:Systems[Bootstrapping].
 
-You can also configure persistent parameters that are used system-wide, including the sudo user.
-For more information on configuring the sudo user, see xref:client-configuration:contact-methods-pushssh.adoc[].
+You can also configure persistent parameters that are used system-wide, including the sudo user to access the system as an unprivileged user instead of as root.
+
+
+
+// FIXME: which commands must we actually list for <user>?
+// FIXME: NOTE probably needs to be adjusted
+.Procedure: Configuring unprivileged SSH access
+. Ensure you have the latest [path]``spacewalk-taskomatic`` and [path]``spacewalk-certs-tools`` packages installed on the {productname} Server.
+. On each client system, create an appropriate unprivileged user.
+. On each client system, edit the [filename]``sudoers`` file:
++
+----
+sudo visudo
+----
+. Grant [command]``sudo`` access to the user by adding this line at the end of the [filename]``sudoers`` file.
+  Replace [systemitem]``<user>`` with the name of the user that is bootstrapping the client in the {webui}:
++
+----
+<user>  ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3, /var/tmp/venv-salt-minion/bin/python
+<user> ALL=(ALL) NOPASSWD:/home/<user>/enable.sh
+<user> ALL=(ALL) NOPASSWD:/home/<user>/bootstrap.sh
```

Comment:
No idea if they are needed. AFAIK salt-ssh onboarding only need the python paths allowed.

Reply:
ok, thanks!